frontend/app/src/people/utils/ExtraFunctions.tsx

```diff
@@ -7,3 +7,12 @@ export const filterCount = (filterValues: any) => {
   }
   return count;
 };
+
+export const formatSat = (sat: number) => {
+  if (sat === 0 || !sat) {
+    return '0';
+  }
+  const satsWithComma = sat.toLocaleString();
+  const splittedSat = satsWithComma.split(',');
+  return splittedSat.join(' ');
+};
```

Comment:
We want to write tests for this but I don't think we have jest working yet

Reply:
> We want to write tests for this but I don't think we have jest working yet

@tobi-bams Why not use this function in helpers.ts ```convertToLocaleString``` instead of creating a new function

Reply:
@elraphty `convertToLocaleString` is returning something else compared to what I want.